packages/webex/src/calling.js

```diff
@@ -47,4 +47,4 @@ class Calling {
   }
 }
 
-export {Calling};
+export default Calling;
```

Comment:
is this also mandatory ?

Reply:
I think [this comment](https://github.com/webex/webex-js-sdk/pull/2902#discussion_r1239333384) will clarify this question too